content/latest/api/ysql/create_sequence.md

```diff
@@ -0,0 +1,162 @@
+
+### start
+<svg class="rrdiagram" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg" width="135" height="35" viewbox="0 0 135 35"><path class="connector" d="M0 22h5m125 0h5"/><rect class="literal" x="5" y="5" width="125" height="25" rx="7"/><text class="text" x="15" y="22">&lt;Integer Literal&gt;</text></svg>
+
+### cache
+<svg class="rrdiagram" version="1.1" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns="http://www.w3.org/2000/svg" width="135" height="35" viewbox="0 0 135 35"><path class="connector" d="M0 22h5m125 0h5"/><rect class="literal" x="5" y="5" width="125" height="25" rx="7"/><text class="text" x="15" y="22">&lt;Integer Literal&gt;</text></svg>
+
+
+### Grammar
+```
+create_sequence = 'CREATE' 'SEQUENCE' [ 'IF' 'NOT' 'EXISTS' ] sequence_name sequence_options;
+sequence_name = '<Text Literal>';
+sequence_options = [ 'INCREMENT' [ 'BY' ] increment ] [ 'MINVALUE' minvalue | 'NO' 'MINVALUE' ] [ 'MAXVALUE' maxvalue | 'NO' 'MAXVALUE' ] [ 'START' [ 'WITH' ] start ] [ 'CACHE' cache ];
+increment = '<Integer Literal>';
+minvalue = '<Integer Literal>';
+maxvalue = '<Integer Literal>';
+start = '<Integer Literal>';
+cache = '<Integer Literal>';
+```
+
+Where
+
+- `sequence_name` is the name of the sequence
+- `increment` is the difference between consecutive values in the sequence. Default is 1.
+- `minvalue` is the minimum value allowed in the sequence. If this value is reached (in a sequence with a negative increment), `nextval()` will return an error. If `NO MINVALUE` is specified, the default value will be used. Default is 1.
+- `maxvalue` is the maximum value allowed in the sequence. If this value is reached, `nextval()` will return an error. If `NO MAXVALUE` is specified, the default will be used. Default is 2^63 - 1;
+- `start` is the first value in the sequence. Default is 1.
+- `cache` how many numbers from the sequence to cache in the client. Default is 1.
+
+## Semantics
+- An error is raised if a sequence with that name already exists in the current schema.
+
+## Cache
+In YSQL as in Postgres, the sequence's data is stored in two values: `last_val` and `is_called`. `last_val` stores the last value allocated for that sequence. When `is_called` is false indicates that `last_val` hasn't been used, and the next call to `nextval()` could return that value.
+By default (when `INCREMENT` is 1), each call to `nextval()` updates `last_val` for that sequence. In YSQL, the table holding the sequence's data is replicated as opposed to being in the local file system. Each update to this table requires two RPCs (a read RPC and a write RPC), so the latency experienced by a single `nextval()` in YSQL will be significantly higher than the same operation in Postgres. To avoid such performance degradation, we recommend using a cache value with a value large enough so that the latency of one allocation can be amortized over all the numbers allocated for the cache.
```

Comment:
Mention that the cached values are stored in memory on the localnode, and avoids the RPC.